public/_data.json

```diff
@@ -1,35 +1,18 @@
 {
     "current": {
-        "title": "",
-        "datetime": "Saturday December 9th, 2017 5pm",
-        "date": "12/18/2017 17:00:00",
-        "host": {
-            "address": "9400 Culver Blvd, Culver City, CA 90232",
-            "name": "Culver Hotel",
-            "misc": "",
-            "url": "http://www.culverhotel.com/"
-        },
-        "drinksjs": {
-            "address": null,
-            "name": null,
-            "misc": null,
-            "url": null
-        },
-        "speakers": [
+        "message": "We are taking off December due to the holidays. We will update the site as soon as the next event is finialized. To get updates on when the next event join us on Slack or subscribe to our mailing list.",
```

Comment:
> We are taking December off due to the holidays.

I'm not sure which is more "correct"

Reply:
I think "taking December off" is more correct. "Taking off December" almost implies the whole js.la team is leaving to go somewhere for the month of December depending on how you read and interpret it. But not a huge deal.

Also, `finalized` is spelled wrong, and `To get updates on when the next event (is) join us` needs to be corrected.